### ISSUE TITLE
Can not request commands from TV in standby

### ISSUE THREAD

**Issue by troykelly** [gerard33/sony_bravia_psk#7]
When requesting `_refresh_commands` ([line](https://github.com/gerard33/sony_bravia_psk/blob/fa8e097d191337c4d24ae77ee597639a8f325d74/braviapsk/sony_bravia_psk.py#L259)) from `get_command_code` ([line](https://github.com/gerard33/sony_bravia_psk/blob/fa8e097d191337c4d24ae77ee597639a8f325d74/braviapsk/sony_bravia_psk.py#L266)) to turn On the TV - an error is generated if the TV is in standby (as would be expected).
```json
{
    "error": [
        7,
        "not power-on"
    ],
    "id": 102
}
```
If the TV is in Standby it's useless calling `get_command_code` anyway as the PowerOn command is not returned for most models of Bravia AFAIK.
The command for supporting TV's would just simply be (turning On a TV in standby)
```json
{
  "method": "setPowerStatus",
  "version": "1.0",
  "params": [
    {
      "status": true
    }
  ],
  "id": 102
}
```

**Comment by gerard33** [comment]
Looks like the same issue as being solved [here](https://github.com/aparraga/braviarc/pull/27)?

**Comment by troykelly** [comment]
No, that doesn't solve the issue.

**Comment by gerard33** [comment]
Should be solved with PR #11.
